packages/core/src/List.js

```diff
@@ -57,14 +59,25 @@ function List({
 
 List.propTypes = {
     variant: PropTypes.oneOf(Object.values(LIST_VARIANTS)),
+    /**
+     * <Section> prop
+     */
```

Comment:
You can also keep it in one line like:
```
/** `<Section>` prop */
```

And don't forget to wrap `<Component>` as inline code or it might be treated as an HTML tag in Markdown parser.

Reply:
Looks great. Thanks!